src/Sulu/Bundle/SecurityBundle/Resources/public/js/models/permission.js

```diff
@@ -1,22 +0,0 @@
-/*
- * This file is part of the Sulu CMS.
- *
- * (c) MASSIVE ART WebServices GmbH
- *
- * This source file is subject to the MIT license that is bundled
- * with this source code in the file LICENSE.
- */
-
-define(['mvc/relationalmodel'], function(relationalModel) {
-
-    'use strict';
-
-    return relationalModel({
-        defaults: function() {
-            return {
-                context: '',
-                permissions: ''
-            };
-        }
-    });
-});
```

Comment:
Why is this file deleted? Does it have to do anything with the tab conditions? Or was it simply not used?